### DISCUSSION TITLE
Line under apps

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#3195] [Q&A]
Any way to remove lines under apps?
![HU2wj4lM5d](https://github.com/valinet/ExplorerPatcher/assets/103335637/6604eb04-2d5a-4205-894e-6b110281fd45)

**Comment by pyrates999** [comment]
No as it is intended to show what apps are opened and what apps are not opened: https://answers.microsoft.com/en-us/windows/forum/all/how-to-remove-the-line-in-taskbar-under-running/389b24cc-2c41-4907-bd04-cabe57eac8a1
